src/js/templates/maps/viewfinder/expansion-panel.html

```diff
@@ -0,0 +1,14 @@
+<div class="<%= classNames.toggle %>">
+  <div class="<%= classNames.icon %>">
+    <i class="icon icon-<%= icon %>"></i>
```

Comment:
there is a naming difference among different versions of fontawesome. the newer versions stopped using `icon-`. let's have the users specify the full class name to help with the migration here.

Reply:
Done